src/Home.js

```diff
@@ -3,6 +3,7 @@ import { useInfiniteQuery } from 'react-query';
 import TinderCard from './libs/react-tinder-card';
 import { fetchRandom } from './api/profiles';
 import Like from './components/like/like';
+import { sendEvent } from './libs/ga-analytics';
```

Comment:
Is it possible to send Social Interaction like in here? https://developers.google.com/analytics/devguides/collection/analyticsjs/social-interactions

Otherwise we can send the sharing action as regular event.

Reply:
sure, we could do `ga('send', 'social', 'Twitter', 'tweet', link_display);`